src/sql/workbench/contrib/notebook/browser/cellViews/linkHandler.directive.ts

```diff
@@ -60,6 +61,15 @@ export class LinkHandlerDirective {
 		} catch {
 			// ignore
 		}
+		// Web mode linking to open files
+		if (isWeb) {
+			// only change scheme for file links (file links in web mode scheme are also http)
+			// therefore we will use the authority to understand if its a local file path
+			if (window.location.host === uri.authority) {
+				uri = uri.with({ scheme: 'vscode-remote' });
+				this.openerService.open(uri);
```

Comment:
Should there be a return statement after here so that it doesn't go to the later cases?